starter/script.js

```diff
@@ -1 +1,57 @@
 // put code here!
+// Shout-out to Rob Cantor, you weirdo
+// If you don't understand what this means, please refer to this video (SFW): https://www.youtube.com/watch?v=o0u4M6vppCI
+
+console.log("You're walking in the woods. There's no one around and your phone is dead. Out of the corner of your eye, you spot him!");
+
+alert("SHIA LABEOUF!");
+
+console.log("He's following you. (About 30 feet back) He gets down on all fours and breaks into a sprint.");
+
+var hesGainingOnYou = prompt("He's gaining on you. What do you do? \n 'R' - run for your life \n 'L' - look for your car");
+
+  if (hesGainingOnYou == 'R') {
+    console.log("It's dark. You seem to have lost him. But you're hopelessly lost yourself, stranded with a murderer.");
+
+    var creepThroughUnderbrush = prompt("How do you proceed through the underbrush? \n 'S' -- silently, to avoid detection \n 'L' -- loudly, to alert others to Shia LaBeouf");
+
+      if (creepThroughUnderbrush = 'S') {
```

Comment:
Careful with your equal signs! You only use one here, which acts as an assignment. When checking a condition, make sure to use either `==` or `===`.